recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,238 @@
+            "texttract",
+            "transcribe",
+            "transfer",
+            "translate",
+            "waf",
+            "workdocs",
+            "worklink",
+            "workmail",
+            "workspaces",
+            "xray"
+           )
+    options = merge_dicts_for_sdk({"build_" + x: [True, False] for x in sdks}, {
+            "shared": [True, False],
+            "fPIC": [True, False],
+            "min_size": [True, False]
+        })
+    default_options = merge_dicts_for_sdk({"build_" + x: False for x in sdks}, {
+            "shared": False,
+            "fPIC": True,
+            "min_size": False
+        })
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.settings.os != "Windows":
+            if self.settings.os != "Macos":
+                self.requires("openssl/1.1.1d")
+            self.requires("libcurl/7.66.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+
+    def build(self):
+        cmake = CMake(self)
+        build_only = list([])
+        for sdk in self.sdks:
+            if getattr(self.options, "build_" + sdk):
+                build_only.append(sdk)
+
+        cmake.definitions["BUILD_ONLY"] = ";".join(build_only)
+        cmake.definitions["ENABLE_UNITY_BUILD"] = "ON"
+        cmake.definitions["ENABLE_TESTING"] = "OFF"
+        cmake.definitions["AUTORUN_UNIT_TESTS"] = "OFF"
+
+        cmake.definitions["MINIMIZE_SIZE"] = "ON" if self.options.min_size else "OFF"
+        cmake.definitions["BUILD_SHARED_LIBS"] = "ON" if self.options.shared else "OFF"
+        cmake.definitions["FORCE_SHARED_CRT"] = "ON" if self.options.shared else "OFF"
+
+        cmake.configure(source_folder=self._source_subfolder, build_folder=self.build_folder)
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        cmake = CMake(self)
+        cmake.install(build_dir=self.build_folder)
+
+    def package_info(self):
+        libs = list([])
+
+        for sdk in self.sdks:
+            if getattr(self.options, "build_" + sdk):
+                libs.append("aws-cpp-sdk-" + sdk)
+        libs.extend(["aws-cpp-sdk-core", "aws-c-event-stream", "aws-c-common", "aws-checksums"])
+
+        if self.settings.os == "Windows":
+            libs.append("winhttp")
+            libs.append("wininet")
+            libs.append("bcrypt")
+            libs.append("userenv")
+            libs.append("version")
+            libs.append("ws2_32")
+
+        if self.settings.os == "Linux":
+            libs.append("atomic")
+            if self.settings.compiler == "clang":
+                libs.append("-stdlib=libstdc++")
```

Comment:
You can use [`tools.stdcpp_library`](https://docs.conan.io/en/latest/reference/tools.html#tools-stdcpp-library).
It can be added to the system libs for all os'es and compilers (not only gcc/clang) as:
```python
lib_stdcpp = tools.stdcpp_library(self)
if lib_stdcpp:
    self.cpp_info.system_libs.append(lib_stdcpp)
```

Reply:
Done here: 71ee769d